File: Data Structures & Algorithms/maximum-subarray/submission-1.cs
public class Solution {
    public int MaxSubArray(int[] nums) {
        int max = nums[0];
        int currentMax = nums[0];

        for(int i = 1; i<nums.Length; i++)
        {
            currentMax = Math.Max(nums[i], currentMax + nums[i]);
            max = Math.Max(max, currentMax);
        }

        return max;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if(len(nums) == 0):
            return 0
        nums_set = set(nums)

        count = 1
        for i in nums_set:
            k = i
            if(i - 1 not in nums_set):
                tmp = 1
                while(k+1 in nums_set):
                    tmp = tmp + 1
                    k = k+1
                count = max(count, tmp)

        return count        